timeseries/src/autogluon/timeseries/dataset/ts_dataframe.py

```diff
@@ -85,9 +85,7 @@ def __init__(self, data: Any, *args, **kwargs):
         elif isinstance(data, Iterable):
             data = self._construct_pandas_frame_from_iterable_dataset(data)
         else:
-            raise ValueError(
```

Comment:
I think this is where black hurts the most. My black version is 22.1 and it doesn't mind this line for example (this is true for _most_ of the changes this PR introduces, but this might just be that I use `blackd` and it doesn't respect the line length config). Still, in my experience, black is also not backward-compatible (maybe they don't even claim to be). 

How are we going to handle this? Should we fix a black version among us (and also in our dedicated env_setup) for use?

Reply:
I haven't discovered backwards compatibility issues with black yet, but fixing the version sounds reasonable.

We can also postpone this decision until the first time that we encounter some backwards-incompatible behavior from a future version of black. The change over here definitely seems related to the `line-length` parameter in the config, can you try running a local version of `black` to see if it causes problems?

Reply:
OK sounds good to me. will try.